### PR TITLE
Hibernate Envers: add support for more configuration properties

### LINKED ISSUE #6112: Support Hibernate Envers configuration properties
**Description**
Currently Envers supports several configuration properties, e.g. `org.hibernate.envers.default_schema`. To me it makes sense to add support for some or all of these properties on the application.properties file.

### PR DESCRIPTION
**Pull request by Sanne**
Adding support for configuring REVISION_FIELD_NAME, REVISION_TYPE_FIELD_NAME, AUDIT_TABLE_SUFFIX, AUDIT_TABLE_PREFIX .

Fixes #6112 , Related to #20831

### CODE REVIEW

**Comment by Sanne** [comment]
Updated as I now understood how to do it properly w/o the odd workarounds.

File: extensions/hibernate-envers/runtime/src/main/java/io/quarkus/hibernate/envers/HibernateEnversBuildTimeConfig.java [hunk 1]
```diff
@@ -8,14 +10,37 @@
...
+     */
+    @ConfigItem(defaultValue = "REV")
+    public Optional<String> revisionFieldName;
+
+    /**
+     * Revision type field name. Defaults to {@literal REVTYPE}.
+     * Maps to {@link org.hibernate.envers.configuration.EnversSettings#REVISION_TYPE_FIELD_NAME}.
+     */
+    @ConfigItem(defaultValue = "REVTYPE")
+    public Optional<String> revisionTypeFieldName;
```
Comment (yrodiere) [inline extensions/hibernate-envers/runtime/src/main/java/io/quarkus/hibernate/envers/HibernateEnversBuildTimeConfig.java:44]: Would you mind adding tests for these? A simple test would be enough, just to check the setting is correctly forwarded to Envers.

I've been bitten by this before: trivial code to forward a setting, but somehow it was not taken into account...

**Review by yrodiere — COMMENTED** [review]
LGTM, though I'd recommend adding tests for the other two settings as well.

**Comment by Sanne** [comment]
I'm adding better tests but it's taking some time as I'm again having issues with the build.

File: extensions/hibernate-envers/runtime/src/main/java/io/quarkus/hibernate/envers/HibernateEnversBuildTimeConfig.java (hunk 1 above)
Comment (Sanne) [reply extensions/hibernate-envers/runtime/src/main/java/io/quarkus/hibernate/envers/HibernateEnversBuildTimeConfig.java:44]: done

**Review by yrodiere — APPROVED** [review]
LGTM, thanks!

**Pull request merged**